vignettes/config.Rmd

```diff
@@ -45,13 +45,19 @@ hugodown does not work with every possible hugo site. There is some config that
     CSS classes. You can generate starter css with (e.g):
     
     ```
-    hugo gen chromastyles --style=pygments
+    hugo gen chromastyles --style=pygments > static/css/pygments.css
```

Comment:
I think it would be better to give this a slightly more generic name, like `highlight.css` or similar.

Reply:
done